content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_upload-certs.md

```diff
@@ -0,0 +1,27 @@
+
+Upload certificates to kubeadm-certs
+
+### Synopsis
+
+This command is not meant to be run on its own. See list of available subcommands.
```

Comment:
Maybe we should update it on the command.

Reply:
for next cycle.
not a critical bug i sort of locked the CLI already by submitting this PR.